File: app/src/main/java/com/my/test2/TitleModel.java
package com.my.test2;

import android.app.Activity;
import android.view.View;

import com.my.test2.databinding.LayoutTitleBinding;

/**
 * @author : yx^_^
 * @e-mail : 565749553@qq.com
 * @date : 2020/3/24 14:15
 * @desc : better than before
 */
public class TitleModel extends SimpleBaseModel<LayoutTitleBinding>{
    public TitleModel(LayoutTitleBinding binding, Activity context) {
        super(binding, context);
    }
    public void setTitle(String str){
        binding.tx.setText(str);
    }
    private void func(){
        //需要重复的业务逻辑
    }
    @Override
    public void onClick(View v) {

    }

}

File: app/src/flavor2/java/com/my/flavor/TitleModel.kt
package com.my.flavor

import android.app.Activity
import android.view.View
import com.my.test2.databinding.LayoutTitleBinding
import com.my.test2.kotlinbase.SimpleBaseModel

/**
 * @author : yx^_^
 * @e-mail : 565749553@qq.com
 * @date : 2020/3/24 14:15
 * @desc : better than before
 */
class TitleModel(binding: LayoutTitleBinding, context: Activity) : SimpleBaseModel<LayoutTitleBinding>(binding,context) {
    var name=""
    fun setTitle() {
        binding.tx.setText("test")

    }

    private fun func() {
    }

    override fun onClick(v: View?) {

    }

}